cmd/policy_ls_groups.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2016-2020, Pulumi Corporation.
```

Comment:
One other thing! The file should be `policy_group_ls.go`.

Reply:
so fast!

Reply:
You just renamed it `policy_ls_group.go`, but it should be `policy_group_ls.go`, right?

Reply:
Now I did 😬